app/controllers/weight_estimator_controller.rb

```diff
@@ -1,5 +1,5 @@
 class WeightEstimatorController < ApplicationController
   def index
-    @household_good_categories = HouseholdGoodCategory.all.order(:name)
+    @household_good_categories = HouseholdGoodCategory.includes(:household_goods).all
```

Comment:
oh right, i remember reading this before when you refactored some other db queries